src/orders/components/OrderDraftDetailsProducts/TableLineForm.tsx

```diff
@@ -56,10 +56,13 @@ const TableLineForm: React.FC<TableLineFormProps> = ({
                 type="number"
                 value={data.quantity}
                 onChange={debounce}
-                onBlur={submit}
-                inputProps={{
-                  min: 1
+                onBlur={async () => {
+                  if (data.quantity < 1) {
+                    await set({ quantity: 1 });
+                  }
+                  submit();
                 }}
+                inputProps={{ min: 1 }}
```

Comment:
```suggestion
                inputProps={{ min: 1, step: 1 }}
```

Reply:
`step: 1` is default, we can be implicit here